.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: |
+      # Install extensions
+      # FIXME: Gitpod uses the Open VSX registry for but the following
+              have not been ported over yet. Once the issues are resolved
+              remove from this task and add to the extensions config below.
+      # https://github.com/cue-sh/vscode-cue/issues/19
+      git clone https://github.com/cue-sh/vscode-cue /workspace/.vscode-remote/extensions/cuelang.org.cue.0.0.1
+      # https://github.com/johnallen3d/vscode-cue-fmt/issues/9
+      git clone https://github.com/johnallen3d/vscode-cue-fmt.git
+
+      # Install CUE and Dagger
+      go install -mod=readonly cuelang.org/go/cmd/cue
+      make install
+
+vscode:
+  extensions:
+    - golang.go
+    - yzhang.markdown-all-in-one
+
+gitConfig:
+  alias.st: status
+  alias.ci: commit
+  alias.co: checkout
```

Comment:
Is there a way to config a sign-off? Dagger contribution guidelines ask that all contributors should signs their commits.

Reply:
I changed the alias to `commit -s` which does the job.